packages/web-components/src/components/feature-card-block-medium/__stories__/feature-card-block-medium.stories.ts

```diff
@@ -14,18 +14,15 @@ import ifNonNull from 'carbon-web-components/es/globals/directives/if-non-null.j
 import readme from './README.stories.mdx';
 import textNullable from '../../../../.storybook/knob-text-nullable';
 import '../feature-card-block-medium';
-import '../feature-card-block-medium-card';
 
 export const Default = ({ parameters }) => {
-  const { blockHeading, heading, defaultSrc, alt, href } = parameters?.props?.['dds-feature-card-block-medium'] ?? {};
+  const { copy, eyebrow, heading, defaultSrc, alt, href } = parameters?.props?.['dds-feature-card-block-medium'] ?? {};
   return html`
-    <dds-feature-card-block-medium>
-      <dds-feature-card-block-medium-heading slot="heading">${blockHeading}</dds-feature-card-block-medium-heading>
-      <dds-feature-card-block-medium-card href=${ifNonNull(href || undefined)}>
-        <dds-image slot="image" alt="${ifNonNull(alt)}" default-src="${ifNonNull(defaultSrc)}"></dds-image>
-        <slot slot="heading">${heading}</slot>
-        ${ArrowRight20({ slot: 'footer' })}
-      </dds-feature-card-block-medium-card>
+    <dds-feature-card-block-medium href=${ifNonNull(href || undefined)}>
+      <slot slot="heading">${heading}</slot>
```

Comment:
`<slot>` is for _defining_ a slot in shadow DOM, instead of for using a slot:

```suggestion
      <span slot="heading">${heading}</span>
```

Reply:
@asudoh Oops, thanks! I forgot to rename the attr to `name` instead of `slot`.